### ISSUE TITLE
[BUG] Exception during login

### ISSUE THREAD

**Issue by rockneverdies55** [RocketChat/Rocket.Chat.Android#1115]
Your Rocket.Chat.Android version: Just installed from the store so whatever is the latest as of today

Mobile device model and OS version: Moto G-Plus - Android 7.1.1

Just trying to log into our server and the app is returning the following exception in a toast message as far as I could read:

`com.squareup.moshi.JsonDataException: Expected an int but was 99999999999 at path $.settings[18].value at $.settings[18].value`

**Comment by rafaelks** [comment]
@rockneverdies55 Looks like you've a setting with the value `99999999999`. Can you change that to a lower number (or empty maybe?) In the meantime, we need to change the number type we're using for the setting.

**Comment by taylorcoffelt** [comment]
I'm having a JSON error on login as well as of version 2.0.1:

```
Use JsonReader.setLenient(true) to accept malformed JSON at path $
```

Android Version 8.1
Google Pixel 2
![screenshot_20180418-084608](https://user-images.githubusercontent.com/690213/38939626-991a7470-42e5-11e8-8068-f741d2598dd5.png)

**Comment by rafaelks** [comment]
@taylorcoffelt Your server looks unavailable to me:

![screen shot 2018-04-18 at 12 05 15](https://user-images.githubusercontent.com/551004/38940560-c9bd0f00-4300-11e8-9a43-94d0d8070558.png)

**Comment by rockneverdies55** [comment]
@rafaelks 
> Looks like you've a setting with the value 99999999999. Can you change that to a lower number (or empty maybe?)

Which value are we talking about?

**Comment by rockneverdies55** [comment]
Ok found it. It was *max file size upload limit* value. I reduced it to 1gb (1073741824) and login was successful.

**Comment by rafaelks** [comment]
@rockneverdies55 Thank you!!! 👍

**Comment by rafaelks** [comment]
This one is fixed in 2.0.2. 👍
